### DISCUSSION TITLE
[bridge] useStore

### DISCUSSION THREAD

**Discussion by Andrevoks** [nuxt/nuxt#18335] [Questions]
I try to migrate from nuxt 2 to nuxt / bridge, removed pakage @ nuxtjs / composition-api, and have trouble with this methods where can I get these methods? useStore from "vuex" doesn't work

**Comment by danielroe** [comment]
To repeat @pi0's comment from issue:

> Hi @Andrevoks. A compatibily layer will be added when using @nuxt/bridge so import { useStore } from '@nuxtjs/composion-api`` should keep working ([src](https://github.com/nuxt/framework/blob/main/packages/bridge/src/runtime/capi.legacy.mjs#L259)). But i advice to change it to getCurrentInstance().proxy.$store` and remove imports.

**Reply by pi0** [reply]
You can also use `getNuxtApp` to migrate. Example: ([codesandbox](https://codesandbox.io/s/winter-glade-xej8n?file=/pages/index.vue:0-157))

```vue
<template>
  <div>Vuex state: {{ state }}</div>
</template>

<script setup>
const { $store } = useNuxtApp().legacyApp;
const state = $store.state;
</script>
```

**Reply by chz** [reply]
@pi0 But not working commit or dispatch, btw i can change directly. 
Check sandbox: https://codesandbox.io/s/sweet-bohr-dqvsq

**Reply by danielroe** [reply]
It seems to work fine: https://codesandbox.io/s/withered-fire-g5z05?file=/pages/index.vue
